### ISSUE TITLE
Search ring resets after exiting venue

### ISSUE THREAD

**Issue by trigger2896** [shnergle/ShnergleApp-iOS#329]
So I am currently in Ealing, if I look at Battersea (which in itself is difficult as the map keeps resetting to my location), then when I look at a venue and then tap back out to around me, the search ring goes straight back to my actual location. Very annoying! :-)

**Comment by krzd** [comment]
The reason why it resets, is because we remove the map if it is not displayed and reinitialize it when coming back to the view. This was/is to prevent to high memory usage, especially whilst sharing/uploading an image.
Question is, if that is still needed with Apple Maps as they might handle memory differently. Will need to try out.

**Comment by krzd** [comment]
So it is still needed, number of memory warnings increases drastically if we do not remove the map. So guess this is a wontfix for now.

**Comment by trigger2896** [comment]
Did this reload happen in 1.4? I don't remember it. I seem to remember the ring stayed where I had left it.
Also, in apple maps the map keeps pinging back to my location when I am trying to put the search ring somewhere. We need to fix both of these before we can roll out apple maps integration as they are too annoying...

**Comment by krzd** [comment]
We introduced that behaviour before the first release I think. That with the location going back is probably related to #330

**Comment by trigger2896** [comment]
I just tried the App Store version on stains phone. The map re-centres, but the search ring doesn't re-centre after viewing a venue. The new version needs to do this too.
